Tests/Tests/CustomEventSpec.swift

```diff
@@ -0,0 +1,50 @@
+import Quick
+import Nimble
+@testable import RInAppMessaging
+import CoreFoundation
+
+class CustomEventSpec: QuickSpec {
+    override func spec() {
+        describe("CustomEvent") {
+            let customAttributes =  [CustomAttribute(withKeyName: "Key1", withBoolValue: false),
+                                     CustomAttribute(withKeyName: "Key2", withDoubleValue: 3),
+                                     CustomAttribute(withKeyName: "Key3", withIntValue: 1)]
+            let customEvent = CustomEvent(withName: "Test4", withCustomAttributes: customAttributes)
+            context("CustomEvent.analyticsParameters") {
+                it("will return non nil dictionary value") {
+                    expect(customEvent.analyticsParameters).toNot(beNil())
+                }
+            }
+            context("CustomEvent.getAttributeMap") {
+                it("will return non nil dictionary value") {
+                    customEvent.customAttributes = customAttributes
+                    expect(customEvent.getAttributeMap()).toNot(beNil())
+                }
+                it("will return nil value") {
```

Comment:
If will return nil value when? You need to add some context. (The previous test states the opposite - that it should return non nil value)